src/worker.ts

```diff
@@ -1,12 +1,21 @@
-import { ping } from './handlers/scheduledEventHandler';
+import { callDiscordNicknameBatchUpdate, ping } from './handlers/scheduledEventHandler';
 import { env } from './types/global.types';
 
 export default {
 	// We need to keep all 3 parameters in this format even if they are not used as as cloudflare workers need them to be present So we are disabling eslint rule of no-unused-vars
 	// for more details read here: https://community.cloudflare.com/t/waituntil-is-not-a-function-when-using-workers-with-modules/375781/4
 	// eslint-disable-next-line no-unused-vars
 	async scheduled(req: Request, env: env, ctx: ExecutionContext) {
-		ctx.waitUntil(ping(env));
+		switch (req.cron) {
+			case '0 */4 * * *':
+				ctx.waitUntil(ping(env));
+				break;
+			case '0 */6 * * *':
+				await callDiscordNicknameBatchUpdate(env);
```

Comment:
if we dont return a response here then how will we generate logs here?

Reply:
Yes, I missed it. I'll return a response here.